src/Customer.js

```diff
@@ -1,8 +1,8 @@
 import User from './User'
 
 class Customer extends User {
-  constructor(id, name) {
-    super(id);
+  constructor(id, name, date, bookings) {
```

Comment:
I think you'll need to move name to the end of your constructor parameters, otherwise super will mess up in calling the User class constructor. No need to change it for this PR, just be aware of it as you start working on Customer.